client/src/Components/Login/login.js

```diff
@@ -0,0 +1,116 @@
+import './login.css';
+import { FaFacebookF } from "react-icons/fa"
+import { FaGithub } from "react-icons/fa"
+import { FaGoogle } from "react-icons/fa"
+import { FaLinkedin } from "react-icons/fa"
+
+
+function login() {
+
+    const signUp = () => {
+        document.querySelector(".container_login").classList.add("sign-up-mode");
```

Comment:
You can use querySelector in react but downside is that you can lose the reference to the DOM element due to Reacts Reactivity. The solution is to use the useRef hook provided by react to store a reference to the DOM element.

Reply:
Thanks Neel for pointing this error but I tried the method you told me to use but due to some reason the code was not function the way it was supposed to thus I implemented querrySelector here. I will still give it a try once again.